tests/dags/good/template-with-invalid-schema-before-do.yaml

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright 2019, Rambler Digital Solutions
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+dags:
+  callback_dag:
+    args:
+      start_date: 2017-07-27
+      schedule_interval: 1d
+    do:
+    - operators:
+        operator_{{ item.name }}:
+          callback: 'tests.utils:{{ item.callback }}'
+          callback_args:
+            '{{ item.args }}'
+      with_items:
+      - name: multi
+        callback: MultiParamOperator
+        args:
+          param1: egg
+          param2: bacon
+          param3: spam
+      - name: simple
+        callback: Operator
+        args:
+          param: egg
+    - operators:
+        operator_{{ item.name }}:
+          class: '{{ item.class }}'
+          args:
+            '{{ item.args }}'
+      with_items:
+      - name: bash
+        class: airflow.operators.bash_operator:BashOperator
+        args:
+          bash_command: 'echo "test"'
+          end_date: 2019-10-25
+    - sensors:
+        sensor_{{ item.name }}:
+          callback: 'tests.utils:{{ item.callback }}'
+      flow:
+        sensor_{{item.name}}:
+        - operator_bash
+        - operator_simple
```

Comment:
This example is rather big, and it is hard to tell which part exactly would have failed in master.

Is it this `flow` which references operators from other `do` blocks or there's more? If none, would it be possible to simplify the test by stripping everything unrelated to the problem being fixed here?